de.metas.business.rest-api/src/main/java/de/metas/rest_api/invoicecandidates/request/JsonInvoiceCandCreateRequest.java

```diff
@@ -55,14 +55,15 @@
 	@JsonInclude(Include.NON_NULL)
 	private LocalDate dateAcct;
 
-	@ApiModelProperty(value = "External reference (document number) on a remote system. Not neccesarily unique, but but the external user will want to filter recrods using it")
+	@ApiModelProperty(value = "External customer's purchase order's documentno. POReference to be set to all invoice candidates, right before enqueueing them.")
 	private String poReference;
 
-	@ApiModelProperty(required = true, value = "Ignore invoice schedule.")
+	@ApiModelProperty(required = true, value = "This is needed when the user wants to invoice sth that is not yet delivered, the DateToInvoice is sometime in the future."
```

Comment:
```suggestion
	@ApiModelProperty(required = true, value = "This is needed when the user wants to invoice candidates that have their `DateToInvoice` sometime in the future."
```

" `DateToInvoice` sometime in the future" can also be the case if also things were delivered